doc/integration/github.md

```diff
@@ -18,6 +18,7 @@ GitHub will generate an application ID and secret key for you to use.
     - Homepage URL: The URL to your GitLab installation. 'https://gitlab.company.com'
     - Application description: Fill this in if you wish.
     - Authorization callback URL: 'https://gitlab.company.com/'
+    - If install from source, default callback URL is '${YOUR_DOMAIN}/import/github/callback'
```

Comment:
I install from source only, cannot know about omnibus path structure, sor.
If structure is same with source installation, I can fix it!